lib/msal-browser/src/naa/mapping/NestedAppAuthAdapter.ts

```diff
@@ -249,4 +265,53 @@ export class NestedAppAuthAdapter {
             return new AuthError("unknown_error", "An unknown error occurred");
         }
     }
+
+    /**
+     *
+     * @param account
+     * @param idToken
+     * @param accessToken
+     * @param reqTimestamp
+     * @returns
+     */
+    public fromCachedTokens(
```

Comment:
Probably should be named something like `makeAuthResult` given that the function expects not just the tokens.

Reply:
Let me check if this is public interface, else can make the change.

Reply:
Done. Mot so great name though - and may be mapping isn't the call. Will tackle finding a new home to this in a later PR.